repo-labels.json

```
@@ -0,0 +1,87 @@
+[
+  {
+    "color": "#eb6420",
+    "description": "This issue or pull request is blocked",
+    "name": "blocked"
+  },
+  {
+    "color": "#000000",
+    "description": "This issue or pull request will cause existing functionality to change",
+    "name": "breaking change"
+  },
+  {
+    "color": "#d73a4a",
+    "description": "This issue or pull request addresses broken functionality",
+    "name": "bug"
+  },
+  {
+    "color": "#07648d",
+    "description": "This issue will be advertised on code.gov's \"Open Tasks\" page (https://code.gov/open-tasks)",
+    "name": "code.gov"
+  },
+  {
+    "color": "#5319e7",
+    "description": "This issue or pull request involves improvements or additions to documentation",
+    "name": "documentation"
+  },
+  {
+    "color": "#cfd3d7",
+    "description": "This issue or pull request already exists",
+    "name": "duplicate"
+  },
+  {
+    "color": "#b005bc",
+    "description": "This issue is an epic",
+    "name": "epic"
+  },
+  {
+    "color": "#0e8a16",
+    "description": "Good for newcomers",
+    "name": "good first issue"
+  },
+  {
+    "color": "#a2eeef",
+    "description": "This issue or pull request will add new or improve existing functionality",
+    "name": "improvement"
+  },
+  {
+    "color": "#fef2c0",
+    "description": "This doesn't seem right",
+    "name": "invalid"
+  },
+  {
+    "color": "#ce099a",
+    "description": "This pull request is ready to merge during the next Lineage kraken release",
+    "name": "Kraken"
+  },
+  {
+    "color": "#fcdb45",
+    "description": "This issue or pull request requires further information",
+    "name": "need info"
+  },
+  {
+    "color": "#a4fc5d",
+    "description": "This pull request is awaiting an action or decision to move forward",
+    "name": "on hold"
+  },
+  {
+    "color": "#ef476c",
+    "description": "Further information is requested",
+    "name": "question"
+  },
+  {
+    "color": "#1d76db",
+    "description": "This issue or pull request relates to pulling in upstream updates",
+    "name": "upstream update"
+  },
+  {
+    "color": "#d4c5f9",
+    "description": "This issue or pull request involves a version bump",
+    "name": "version bump"
+  },
+  {
+    "color": "#ffffff",
+    "description": "This issue will not be worked on ",
```

Remove an unintended space:
```suggestion
    "description": "This issue will not be worked on",
```